.github/scripts/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+dos2unix buildcoresworking.sh
+dos2unix cleanandbuildworking.sh
+
+# TODO: move into separate repo with assets
+wget https://raw.githubusercontent.com/EricGoldsteinNz/SF2000_Resources/main/OS/V1.6/bios/bisrv.asd -O /__w/sf2000_multicore/sf2000_multicore/bisrv_08_03.asd
```

Comment:
@madcock this should probably be either moved within/outside this repo, or we should get an approval from Eric to use this dump :x 
LMK which one you'd prefer